lib/kafka/consumer.rb

```diff
@@ -217,7 +225,7 @@ def each_message(min_bytes: 1, max_bytes: 10485760, max_wait_time: 1, automatica
       )
 
       consumer_loop do
-        batches = fetch_batches
+        batches = @interceptors.on_consume(batches: fetch_batches)
```

Comment:
as most of the time for more users the list of the interceptors pipeline will be empty, I think we can add a `is_empty?` check on the interceptors pipeline to avoid making this call